socs/agents/hwp_supervisor/agent.py

```diff
@@ -182,6 +236,8 @@ def from_args(cls, args: argparse.Namespace):
             self.gripper_iboot = IBootState(args.gripper_iboot_id, args.gripper_iboot_outlets)
         if args.driver_iboot_id is not None:
             self.driver_iboot = IBootState(args.driver_iboot_id, args.driver_iboot_outlets)
+        if args.acu_instance_id is not None:
```

Comment:
I would want to see log messages for each of these (not just ACU):
"Checking ACU elevation range is [NOT active / active at instance_id={instance_id}]"
etc.

When something goes wrong, whether the agent though these things were on, or not, is very useful log output.